returns/primitives/container.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, inner_value) -> None:
         """
         object.__setattr__(self, '_inner_value', inner_value)  # noqa: WPS609
 
-    def __str__(self) -> str:
+    def __repr__(self) -> str:
         """Converts to string."""
```

Comment:
Can you update the docstring, please?
`__repr__` is more than a simple string conversion.

Reply:
Yes I will change it give me a minute. I had kept it as """Used to display details of the object""" before. Is that fine?

Reply:
👍